app/inbound/dax/validate-dax.js

```diff
@@ -0,0 +1,15 @@
+const schema = require('./schema')
+
+const validateDax = (dax, paymentReference) => {
+  const result = schema.validate(dax, {
+    abortEarly: false
+  })
+
+  if (result.error) {
+    throw new Error(`Payment request with paymentReference: ${paymentReference} does not have the required DAX data: ${result.error.message}`)
```

Comment:
Error message should be on Dax, but not payment request

Reply:
Removed payment request text in favour of Dax